public/app/features/expressions/components/Reduce.tsx

```diff
@@ -66,8 +72,30 @@ export const Reduce = ({ labelWidth = 'auto', onChange, refIds, query }: Props)
     );
   };
 
+  const strictModeNotification = () => {
+    if (mode !== ReducerMode.Strict) {
+      return null;
+    }
+    return (
+      <Alert title={t('reduce.strictMode.title', 'Strict Mode Behaviour')} severity="info">
+        <Trans i18nKey="reduce.strictMode.description">
```

Comment:
You need to add the key in this file https://github.com/grafana/grafana/blob/main/public/locales/en-US/grafana.json
And then run `make i18n-extract` then commit the files.

Reply:
Added the translations keys, an incorrect rebase which I now reverted, made the grafana-pr-automation add extra tags to the issue. I don't think I have the permission to remove these. 
@itsmylife Can you please review?